eox_nelp/stats/views.py

```diff
@@ -47,4 +47,4 @@ def get_tenant_stats(request):
     context = {query_param: "true" for query_param in STATS_QUERY_PARAMS}
     context.update(request.GET.dict())
 
-    return render_to_response("tenant_stats.html", context)
+    return edxmako.shortcuts.render_to_response("tenant_stats.html", context, "main", request)
```

Comment:
Why is need the arg "main" to this render_to_response method,

Reply:
Ok I think this is related to that main dirs or something to do it with that way.
https://github.com/eduNEXT/eox-nelp/pull/98/files#diff-ec9dfd84694298d271197a0909b7b62e0301e9df3dce5c29d0cec0a8255ebd42L19
https://github.com/eduNEXT/eox-nelp/blob/13ed2ffd30202ead25fb67dd7f798b1c6c0f8085/eox_nelp/templates_config.py#L19